apps/sms/index.html

```diff
@@ -608,13 +608,15 @@ <h1 class="message-subject">
 
     <div id="messages-group-header-tmpl" class="hide">
       <!--
-      <bdi>${name}</bdi><bdi dir="ltr"> (+${participantCount})</bdi>
+      <span class="group-header-title">
```

Comment:
Maybe we can just put ```&nbsp;``` between these two ```<bdi>``` and get rid of all these ```.count``` styles, what do you think? + IIRC gaia-header uses ```textContent.length``` to determine length of the title when it's auto adjusting font size. Margin will be ingored, while ```&nbsp;``` will be considered as whitespace char.